src/common/com/intellij/plugins/haxe/ide/annotator/HaxeSemanticAnnotator.java

```diff
@@ -123,31 +133,50 @@ public static void checkProperty(final HaxeFieldModel field, final AnnotationHol
       holder.createErrorAnnotation(field.getSetterPsi(), "Invalid setter accessor");
     }
 
-    if (field.getGetterType() == HaxeAccessorType.GET) {
-      final String methodName = "get_" + field.getName();
-      HaxeMethodModel method = field.getDeclaringClass().getMethod(methodName);
-      if (method == null) {
-        Annotation annotation = holder.createErrorAnnotation(field.getGetterPsi(), "Can't find method " + methodName);
-        annotation.registerFix(new HaxeFixer("Add method") {
-          @Override
-          public void run() {
-            field.getDeclaringClass().addMethod(methodName);
+    ResultHolder fieldType = field.getResultType();
+
+    final HaxeClassModel clazz = field.getDeclaringClass();
+
+    if (!clazz.isExternOrInterface()) {
+      if (field.getGetterType() == HaxeAccessorType.GET) {
+        final String methodName = "get_" + field.getName();
```

Comment:
I have created a couple of methods at HaxeFieldModel to get expected getter setter method names.
https://github.com/TiVo/intellij-haxe/commit/783950b56f7ab220feec61b77acc1359eb876137